cms/src/components/languages/CreateOrEditLanguageModal.vue

```diff
@@ -22,29 +24,37 @@ const emit = defineEmits(["close", "created", "updated"]);
 // Check if we are in edit mode (if a language is passed)
 const isEditMode = computed(() => !!props.language);
 
+const canEditOrCreate =
+    verifyAccess(["group-private-editors"], DocType.Language, AclPermission.Edit) ||
+    verifyAccess(["group-private-editors"], DocType.Language, AclPermission.Edit);
```

Comment:
We should never use hard-coded references to data that might not be available in the database in production code. (In tests it is obviously ok and needed.)